src/sapi.rs

```diff
@@ -18,147 +23,124 @@ use std::time::Instant;
 
 use crate::window::*;
 
-pub const WM_SAPI_EVENT: minwindef::UINT = winuser::WM_APP + 15;
-pub const WM_APP_NOTIFICATION_ICON: minwindef::UINT = winuser::WM_APP + 16;
+pub const WM_SAPI_EVENT: u32 = wm::WM_APP + 15;
+pub const WM_APP_NOTIFICATION_ICON: u32 = wm::WM_APP + 16;
 
-pub struct Com {
-    hr: winnt::HRESULT,
-}
+pub struct Com {}
 
 impl Com {
     pub fn new() -> Com {
         println!("new for Com");
-        // https://msdn.microsoft.com/en-us/library/windows/desktop/ms678543.aspx
-        let hr = unsafe { winapi::um::objbase::CoInitialize(null_mut()) };
-        if failed(hr) {
-            panic!("failed for Com");
+        match unsafe { syscom::CoInitialize(Some(null_mut())) } {
+            Ok(_) => Com {},
+            Err(_) => panic!("failed for Com"),
         }
-        Com { hr }
     }
 }
 
 impl Drop for Com {
     fn drop(&mut self) {
-        // https://msdn.microsoft.com/en-us/library/windows/desktop/ms688715.aspx
-        if self.hr != winapi::shared::winerror::RPC_E_CHANGED_MODE {
```

Comment:
Does this case actually need to be handled? I'm not sure this return code can ever occur here, but I'm not sure.